core/domain/html_cleaner_test.py

```diff
@@ -158,10 +158,12 @@ def test_get_rte_components(self):
         test_data = (
             '<p>Test text&nbsp;'
             '<oppia-noninteractive-math '
-            'raw_latex-with-value="&amp;quot;\\frac{x}{y}&amp;quot;">'
+            'raw_latex-with-value="&amp;quot;\\\\frac{x}{y}&amp;quot;">'
```

Comment:
The problem arose here. Note that there should be two backslashes before frac -- this is what is created in the frontend. This should be represented in Python as four backslashes, not two.

Reply:
Aren't there frontend changes required for this then? Since, when I had manually created `\frac`, it was represented as only `\\frac` in the string here.

Reply:
It is indeed \\frac in the backend (you can see this if you download the yaml file). But in Python, when you write a string with two backslashes, you need to type 4 \ characters.

Reply:
Ok, so the frontend sends `\\frac` to the backend, but  to represent that in the test, we need to add `\\\\frac`, right? So, in normal working, does python add 2 backslashes to the received text so that it is read as 2 backslashes and not combined into 1?

Reply:
The first sentence is correct, yes.

Python doesn't add anything to the received text. It's just that, to represent a single backslash char in a string in Python, you need to type \\. That's because, usually, \ escapes the next char in the string (e.g. \n is just 1 char).